doc/rados/operations/pools.rst

```diff
@@ -96,60 +102,49 @@ Where:
 
 .. describe:: {pg-num}
 
-   The total number of placement groups for the pool. See :ref:`placement
-   groups` for details on calculating a suitable number. The default value of
-   :confval:`osd_pool_default_pg_num` is likely too small for production pools
-   used for bulk data, including RBD and RGW data and bucket pools
-   respectively.
+   The total number of PGs in the pool. For details on calculating an
+   appropriate number, see :ref:`placement groups`. The default value ``8`` is
+   NOT suitable for most systems.
 
   :Type: Integer
-  :Required: No. Set to ``1`` if autoscaling is enabled, otherwise picks up Ceph
-         configuration value :confval:`osd_pool_default_pg_num`
-  :Default: Value of :confval:`osd_pool_default_pg_num`
+  :Required: Yes.
+  :Default: 8
 
 .. describe:: {pgp-num}
 
-   The total number of placement groups for placement purposes. This
-   **should be equal to the total number of placement groups**, except
-   for placement group splitting scenarios.
+   The total number of PGs for placement purposes. This
+   **should be equal to the total number of PGs**, except
+   in PG-splitting scenarios.
 
   :Type: Integer
-  :Required: No. Picks up Ceph configuration value :confval:`osd_pool_default_pgp_num`
-         if not specified. If that is not set, defaults to value of ``pg-num``.
-  :Default: Value of ``pg-num`` 
+  :Required: Yes. If no value has been specified in the command, then the default value is used (unless a different value has been set in Ceph configuration).
+  :Default: 8
 
 .. describe:: {replicated|erasure}
 
-   The pool type which may either be **replicated** to
-   recover from lost OSDs by keeping multiple copies of the
-   objects or **erasure** to get a kind of
-   `generalized RAID5 <../erasure-code>`_ capability.
-   The **replicated** pools require more
-   raw storage but implement all Ceph operations. The
-   **erasure** pools require less raw storage but only
-   implement a subset of the available operations.
+   The pool type. This can be either **replicated** (to recover from lost OSDs
+   by keeping multiple copies of the objects) or **erasure** (to achieve a kind
+   of `generalized RAID5 <../erasure-code>`_ capability).  The **replicated**
+   pools require more raw storage but can implement all Ceph operations. The
+   **erasure** pools require less raw storage but can implement only some Ceph
+   operations.
```

Comment:
Accepted with minor modifications.

Reply:
I had also wondered if this was still true, or if this was true of an earlier, more prototype-like version of erasure pools.

Reply:
Yep.  At first RBD pools couldn't be EC because of overwrites, but that was resolved with Kraken or Luminous.  Today metadata pools still have to be replicated, including I think RGW index.